content/en/docs/concepts/security/index.md

```diff
@@ -118,13 +116,13 @@ work together with `istiod` to automate key and certificate
 rotation at scale. The following diagram shows the identity
 provisioning flow.
 
-{{< image width="75%"
+{{< image width="40%"
     link="./id-prov.svg"
-    caption="Identity Provision"
+    caption="Identity Provisioning Workflow"
     >}}
 
-Istio provisions identities through the
-[secret discovery service (SDS)](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret#secret-discovery-service-sds)
+Istio provisions keys and certificates through the
+[Envoy secret discovery service (SDS)](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret#secret-discovery-service-sds)
```

Comment:
Do we need to mention Envoy? Or SDS is enough?

Reply:
Mentioning Envoy here seems good.